src/azul/service/manifest_service.py

```diff
@@ -919,4 +919,27 @@ def qualify(qualifier, column_name, index=None):
                             assert False
             # Join concatenated values using the joiner
             row = {k: self.column_joiner.join(sorted(v)) if isinstance(v, set) else v for k, v in row.items()}
+            sanitize_participant_id(row)
```

Comment:
Don't like the "patch" after the fact approach and that it would even remove colons from the UUID part. This should be more surgical in only translating the version part and it should explain where the colons come from. It should also be applied where the row is composed, not after everything is done.